sdk/dotnet/Group.cs

```diff
@@ -48,7 +48,7 @@ namespace Pulumi.AzureAD
     ///         {
     ///             DisplayName = "J Doe",
     ///             Password = "notSecure123",
-    ///             UserPrincipalName = "j.doe@terraform.onmicrosoft.com",
+    ///             UserPrincipalName = "jdoe@hashicorp.com",
```

Comment:
Replace hashi with pulumi? :)

Reply:
I will take a fork next time around and remove this...